plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala

```diff
@@ -291,34 +291,24 @@ object ScalafmtPlugin extends AutoPlugin {
 
   private lazy val sbtSources = Def.task {
     val rootBase = (LocalRootProject / baseDirectory).value
-    val thisBase = (thisProject.value).base
     val rootSbt =
-      BuildPaths.configurationSources(thisBase).filterNot(_.isHidden)
+      BuildPaths.configurationSources(rootBase).filterNot(_.isHidden)
     val metabuildSbt =
-      if (rootBase == thisBase)
```

Comment:
I am still trying to figure out in which case this could have been `false` - git blame didn't really help so far